app/Http/Resources/VictimResource.php

```diff
@@ -23,11 +23,14 @@ public function toArray(Request $request): array
             'x_handle' => $this->x_handle,
             'photo_url' => $this->photo_url,
             'status' => $this->status,
-            'holding_location' => $this->holding_location?->name,
```

Comment:
@Marshud the ? is the null safe operator, the rest is not evaluated if that value is null - you will get errors if its removed

Reply:
Duly noted